experimental/canvas/explorations-py/login-access-data-exploration/README.md

```diff
@@ -22,27 +22,8 @@ A capture of these data points, or an assessment that these are not available
 
 ## Getting Started
 
-1. If you don't already have it, install
-   [pipenv](https://docs.pipenv.org/install). (!) WARNING! pipenv does not like
-   it if you have multiple versions of Python installed. You may run into errors
-   in that case.
-
-    ```powershell
-    pip install --user pipenv
-    ```
-
-2. And add this path to your PATH environment variable (assuming you are running
-   Python 3.8): `%APPDATA%\Python\Python38\scripts`.
-
-3. (!) If on the MSDF network, first get a copy of the MSDF root certificate and
-   place it in your `c:\`, then run the following in your PowerShell window,
-   otherwise the `pipenv install` command will fail.
-
-    ```powershell
-    $env:REQUESTS_CA_BUNDLE="C:\msdfrootca.cer"
-    ```
-
-4. Now run `pipenv install` to install required modules.
-
-5. Open `explore.ipynb` in your favorite Jupyter platform and run it.
-
+1. In the /explorations-py folder, run poetry install.
+2. Select the generated virtual environment as interpreter in the notebook. In VSCode you only have to click the python version in the upper right corner, and it should show the option for selecting the virtual environment.
```

Comment:
Please re-wrap this line so that it will be easier to read in VS Code. I like the `Rewrap` extension in VS Code to give me a quick command (Control-Q).

Reply:
In my case, it worked with `alt`+`Q`... I'll run it in all readme files.